src/StoreKeeper/WooCommerce/B2C/Commands/SyncWoocommerceSingleProductImage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Commands;
+
+use Exception;
+use StoreKeeper\WooCommerce\B2C\Exceptions\BaseException;
+use StoreKeeper\WooCommerce\B2C\Imports\ProductImageImport;
+use StoreKeeper\WooCommerce\B2C\Tools\TaskHandler;
+
+class SyncWoocommerceSingleProductImage extends SyncWoocommerceProductPage
```

Comment:
```suggestion
class SyncWoocommerceSingleProductAllImages extends SyncWoocommerceProductPage
```

Reply:
This no longer exist, as images are imported during ProductImport as well.